install-dev/upgrade/sql/1.7.0.0.sql

```diff
@@ -81,40 +81,32 @@ ALTER TABLE  `PREFIX_product_lang` ADD  `social_sharing_description` VARCHAR( 25
 
```

Comment:
Line 76 (i can point this), same as last_connection_date : 

ALTER TABLE `PREFIX_product_attribute_shop` CHANGE `available_date` `available_date` DATE NULL DEFAULT NULL;
ALTER TABLE `PREFIX_product_attribute` CHANGE `available_date` `available_date` DATE NULL DEFAULT NULL;

and report it on db_structure.sql lines : 1512 & 1532